source/faq.txt

```diff
@@ -41,6 +41,25 @@ For applications that require a mix of the new and legacy APIs, ``com.mongodb.Mo
 - Configuration with ``MongoClientSettings`` and ``ConnectionString``, the only difference being that you create instances via constructors instead of a factory class.
 - CRUD API using ``MongoDatabase``, and from there, ``MongoCollection``.  You can access this API via the ``getDatabase()`` method.
 
+How do I use the ``MongoClientSettings`` class?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``MongoClientSettings`` class lets you specify configurations for
+``MongoClient`` instances. The ``MongoClientSettings`` class is immutable. To
+construct ``MongoClientSettings`` instances, use the
+``MongoClientSettings.Builder`` class. Here are the sections of our
+documentation that discuss aspects of the ``MongoClientSettings`` class:
```

Comment:
suggestion: make this more user action-oriented
comment: the suggestion removes the short sentence "it is immutable" as I felt it didn't add anything to the answer.

```suggestion
The ``MongoClientSettings`` class is used to specify configurations for
``MongoClient`` instances. Use the ``MongoClientSettings.Builder`` class 
to construct ``MongoClientSettings`` instances. Here are the sections of our
documentation that discuss aspects of the ``MongoClientSettings`` class:
```

Reply:
Thanks for the note. I agree this should be made more task oriented, and I agree that mentioning immutability is not necessary for readers.

I redid the list of linked content to be task based, and redid the introductory paragraph to focus on what the reader's actions. Fix: https://github.com/mongodb/docs-java/pull/117/commits/295ead332d281e071291e3725a0c5726f68a6706

Going forward, I will continue to refine my sense for which details are helpful and relevant, and which are interesting but should be omitted (in this case immutability of `MongoClientSettings`).